implementation/src/test/java/io/smallrye/mutiny/operators/UniOnItemInvokeTest.java

```diff
@@ -80,7 +101,23 @@ public void testAsyncInvokeOnItem() {
     }
 
     @Test
-    public void testAsyncInvokeOnFailure() {
+    public void testInvokeUniOnItemWithShortcut() {
+        AtomicInteger res = new AtomicInteger();
+        AtomicInteger twoGotCalled = new AtomicInteger();
+
+        int r = one.onItem().invokeUni(i -> {
```

Comment:
Same here, I'd add the test with `one.invokeUni`

Reply:
And same here!